client/src/TreeMap/TreeMapSidebar.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+import { TreeMapControls } from "./TreeMapControls";
+import "./TreeMap.scss";
+
+export interface LocationProps {
+  hash: string;
+  pathname: string;
+  search: string;
+}
+
+export interface HistoryProps {
+  action: string;
+  block: Function;
+  createHref: Function;
+  go: Function;
+  goBack: Function;
+  goForward: Function;
+  length: number;
+  listen: Function;
+  location: LocationProps;
+  push: (arg: string) => void;
+  replace: Function;
+}
```

Comment:
Should be able to get these types from `@types/react-router`, we've got that in our package.json already.